### PR TITLE
Add step evaluation to ptg eval

### PR DESCRIPTION
**Pull request by hdefazio**

### CODE REVIEW

**Comment by hdefazio** [comment]
Depends on #262

File: angel_system/data/common/load_data.py [hunk 1]
```diff
@@ -174,14 +187,292 @@ def activities_from_dive_csv(filepath: str) -> List[Activity]:
...
+        step_seq.append(
+            Step(
+                current_step_id,  # current id
+                sanitize_str(current_step),  # class_label
+                latest_sensor_ts,  # start
+                latest_sensor_ts,  # end
+                conf_vec[current_step_id],  # conf
+                completed_vec[current_step_id],  # completed
+            )
+        )
```
Comment (Purg) [inline angel_system/data/common/load_data.py:472]: Not every message is a unique step, the output here is a streaming state message. I believe you will need to detect transitions if you want to have your output be discrete step time volumes.

**Review by Purg — COMMENTED** [review]
I don't know if I'm specifically confident steps and activities can be scored the same way, but I suppose you are on the trajectory to trying it out and seeing what parts are applicable and make sense, and what doesn't.

File: angel_system/data/common/load_data.py (hunk 1 above)
Comment (Purg) [reply angel_system/data/common/load_data.py:474]: OBE

**Pull request merged**